### PR TITLE
Tests/QUnit: Fix - Mirroring.

### PR DESCRIPTION
**Pull request by iNewLegend**

### CODE REVIEW

File: tests/qunit/tests/assets/dev/js/editor/document/elements/commands/import.spec.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import ElementsHelper from '../../elements/helper';
+import ElementsHelper from '../helper';
 import HistoryHelper from '../../history/helper';
-import BlockFaq from 'elementor/tests/qunit/mock/library/blocks/faq';
-import PageLandingPageHotel from 'elementor/tests/qunit/mock/library/pages/landing-page-hotel';
+import BlockFaq from 'elementor-tests-qunit/mock/library/blocks/faq.json';
```
Comment (matipojo) [inline tests/qunit/tests/assets/dev/js/editor/document/elements/commands/import.spec.js:3]: Remove all  `elementor-tests-qunit` aliases.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: tests/qunit/tests/assets/dev/js/editor/document/elements/commands/import.spec.js (hunk 1 above)
Comment (iNewLegend) [reply tests/qunit/tests/assets/dev/js/editor/document/elements/commands/import.spec.js]: Handled.

File: karma.conf.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ module.exports = function( config ) {
 			'assets/lib/backbone/backbone.radio.min.js',
 
 			// Elementor Common.
-			'tests/qunit/setup-elementor-common.js',
+			'tests/qunit/setup/setup-elementor-common.js',
```
[the remark this answers is not in the document]
Comment (iNewLegend) [reply karma.conf.js]: Handled.

File: tests/qunit/tests/assets/dev/js/editor/container/container.spec.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import ElementsHelper from 'elementor-tests-qunit/core/editor/document/elements/helper';
+import ElementsHelper from 'elementor/tests/qunit/tests/assets/dev/js/editor/document/elements/helper';
```
Comment (matipojo) [inline tests/qunit/tests/assets/dev/js/editor/container/container.spec.js:1]: Remove the alias `elementor-tests-qunit` from alises list.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: tests/qunit/tests/assets/dev/js/editor/container/container.spec.js (hunk 3 above)
Comment (iNewLegend) [reply tests/qunit/tests/assets/dev/js/editor/container/container.spec.js:1]: Handled.

**Review by matipojo — APPROVED** [review]

**Review by kobizz — APPROVED** [review]

**Review by KingYes — APPROVED** [review]

**Pull request merged**